samples/ironside_se/secondary_boot_gen_uicr/secondary/secondary_partition.overlay

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/ {
+	chosen {
+		zephyr,code-partition = &secondary_partition;
```

Comment:
not for this PR but in future you could (optionally) change this so sysbuild applies this automatically to the image